test/BinarySearch.java

```diff
@@ -0,0 +1,56 @@
+//Contributed by Jasani Dhruv.
+//Binary Search Algo.
+
+import java.util.Scanner;
```

Comment:
Looks good! However, you may have seen this program in a few other pull requests (#1634, #1673). We don't currently support it. If you'd like this added to the repo, you'd need to write up the documentation for it.